### PR TITLE
added moonlight Palette

### PR DESCRIPTION
**Pull request by Abod1960**
# Pull Request

## Description

https://github.com/Schrodinger-Hat/ImageGoNord/labels/new-palette

@moonlight-theme is a dark color theme inspired by the camping vibe 🏕
and what I did is basically adding it to the palettes, I tested this locally*...

## Type of change

- [x] New feature (non-breaking change which adds functionality)

### Moonlight

moonlight

**Test Configuration**:
* Firmware version:
* Hardware:
* Toolchain:
* IGN Version:

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [x] I have checked my code and corrected any misspellings

### CODE REVIEW

**Review by Wabri — CHANGES_REQUESTED** [review]
Could you please rename the files without spaces?

Night sky.txt => NightSky.txt
White moon.txt => WhiteMoon.txt

Btw awesome palette 🚀

**Comment by Abod1960** [comment]
> Could you please rename the files without spaces?
> 
> Night sky.txt => NightSky.txt White moon.txt => WhiteMoon.txt
> 
> Btw awesome palette 🚀

done 💠

File: reddit-posts-automation/auto-publish-multipalette.py [hunk 1]
```diff
@@ -31,6 +31,7 @@
   # 'TOKYO': {'id': 'ccc6e9f2-0e43-11ec-9715-1ea24e7b246f', 'hex_colors': ["#1a1b26","#4e5173","#F7768E","#F7768E","#9ECE6A","#9ECE6A","#E0AF68","#E0AF68","#7AA2F7","#7AA2F7","#9a7ecc","#9a7ecc","#4abaaf","#4abaaf","#acb0d0","#acb0d0"]},
   'VAPORWAVE': {'id': 'd84eab48-0e43-11ec-811e-4edcb0b81b7f', 'hex_colors': ["#94D0FF", "#8795E8", "#966bff", "#AD8CFF", "#C774E8"]},
   'CATPPUCCIN': {'id': 'e62951d2-7a09-11ec-abde-ca9aae39851e', 'hex_colors': ["#F5E0DC", "#F2CDCD", "#DDB6F2", "#F5C2E7", "#E8A2AF", "#F28FAD", "#F8BD96", "#FAE3B0", "#ABE9B3", "#B5E8E0", "#96CDFB", "#89DCEB", "#C9CBFF", "#161320","#1A1826","#1E1E2E","#302D41","#575268","#6E6C7E","#988BA2","#C3BAC6","#D9E0EE"]},
+  'MOONLIGHT': {'id': '', 'hex_colors': ["#c0cedd", "#99a0a8", "#c6c3c0", "#191f28", "#1f2730", "#374656", "#ecbd7c", "#c69f68", "#93764d", "#c68768", "#c66868", "#c66890", "#6880c6","#68aec6","#9fc76d"]},
```
Comment (Wabri) [inline reddit-posts-automation/auto-publish-multipalette.py:34]:
```suggestion
  'MOONLIGHT': {'id': '9bbf781a-2cfb-11ed-a549-1ac2767c974c', 'hex_colors': ["#c0cedd", "#99a0a8", "#c6c3c0", "#191f28", "#1f2730", "#374656", "#ecbd7c", "#c69f68", "#93764d", "#c68768", "#c66868", "#c66890", "#6880c6","#68aec6","#9fc76d"]},
```

**Review by Wabri — CHANGES_REQUESTED** [review]
I add a suggestion for flair id of reddit 😁

**Review by Wabri — APPROVED** [review]

**Pull request merged**